### PR TITLE
new test

### PR DESCRIPTION
**Pull request by isabelrios**
Adding a new test for the GET /api/v1/images end point.
For the rest I will wait to have them properly working.

### CODE REVIEW

**Comment by isabelrios** [comment]
Hi @samgiles asking for r? for this first test. 
Thanks!

File: test/server_test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+const chakram = require('chakram'), expect = chakram.expect;
+const fs = require('mz/fs');
+
+describe('Photo-wall server tests', function() {
+
+  const URL = 'http://localhost:3003';
+  var listImages;
```
Comment (samgiles) [inline test/server_test.js]: NIT: prefer `let` - `es-lint` rules have no-var at the moment.

File: test/server_test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+const chakram = require('chakram'), expect = chakram.expect;
+const fs = require('mz/fs');
```
Comment (samgiles) [inline test/server_test.js:3]: Missing `mz` from package.json?

File: test/server_test.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+describe('Photo-wall server tests', function() {
+
+  const URL = 'http://localhost:3003';
+  var listImages;
+  var imagesStored;
```
Comment (samgiles) [inline test/server_test.js]: NIT: as above about `let`

**Comment by samgiles** [comment]
Looks good! r+ with nits addressed.

File: test/server_test.js (hunk 2 above)
Comment (isabelrios) [inline test/server_test.js:3]: Added!

**Comment by isabelrios** [comment]
Thanks for the review!comments addressed. Merging...

**Pull request merged**